### DISCUSSION TITLE
Parts recommendations?

### DISCUSSION THREAD

**Discussion by phillyaws** [jon-harper/OmniBox#83] [Q&A] [answered]
Knocking together my first OmniBox, everything's going brilliantly aside from a few "No one to blame but myself" things, but have one question:
Are there any specific panel mount USB headers you've test-fit with?  I'm looking for USB-C, and USB-micro panel headers, specifically.

Also, made a 4-panel keystone side panel!

**Comment by jon-harper** [comment, marked as answer]
This is a phenomenally good question because it reminds me of something I didn't do. [The list of supported panel mounts](https://jon-harper.github.io/OmniBox/0.9.9/support/panel_mounts/#supported-hardware) has the info you need. I meant to put a reference and link in the BOM and...womp womp. Thanks for the reminder!

Caveat: I haven't tested a MicroUSB extension, so there isn't a predesigned cutout for one. The cutouts for USB panels are standardized, though, so if you make one, it will work with any brand.

> Also, made a 4-panel keystone side panel!

Oh sweet! I'd love to see that. I haven't written up the contrib guidelines yet, but all contributions will be credited in the docs if you want to share. 🙂

**Comment by phillyaws** [comment]
> Oh sweet! I'd love to see that. I haven't written up the contrib guidelines yet, but all contributions will be credited in the docs if you want to share. 🙂

Am I good to upload it to Printables, or would you prefer some other method?

**Comment by jon-harper** [comment]
I will take some time in the nearish future to set contribution guidelines
to make this smoother.

Printables will ensure you are properly attributed for your work, so that
works for me. 🙂

**Reply by phillyaws** [reply]
Uploaded!
[OmniBox - Raspberry Pi 4 Tray with SD Card slot
](https://www.printables.com/model/341728-omnibox-raspberry-pi-4-tray-with-sd-card-slot)

[OmniBox - Side Plate with 4 Keystone Jacks
](https://www.printables.com/model/341726-omnibox-side-plate-with-4-keystone-jacks)

**Reply by jon-harper** [reply]
Cool! Two quick notes about keystones: 

1. I updated the keystone jack STEP with a 1.6mm fillet pretty recently. It makes insertion and removal easier and would be a quick modification:

![image](https://user-images.githubusercontent.com/11852387/208237580-6b31772e-585c-40a3-8fd0-06a1c6375389.png)

2. There's a slight artifact in your side plate where one of the jacks joins. Hope this helps if you're having any issues.

![image](https://user-images.githubusercontent.com/11852387/208237613-f69a0392-7fdf-44f9-9b24-327d4b53e558.png)

Thanks again! I'll add links to the docs over the coming week.

**Reply by phillyaws** [reply]
Good point, and good looking out!  I'll correct them and re-upload.
